packages/core/src/setup.ts

```diff
@@ -36,6 +37,9 @@ type Options = {
 }
 
 export const setup = async (options: Options) => {
+  defaultLogger.level = getLogLevel(options.runtimeLogLevel)
```

Comment:
this has nothing to do with chopsticks logger. this is runtime logger

Reply:
it is set here https://github.com/AcalaNetwork/chopsticks/pull/390/files#diff-42b25a5fdf620c3a1f0a17037770d594bb96251313e608d51a60eac9f2a6637eR109
This features should add a RPC call to change this value

Reply:
got it, @xlc has messaged me as well